### ISSUE TITLE
Crane 1.3.0: Unable to save Docker credentials

### ISSUE THREAD

**Issue by DaveFlynn** [InfuseAI/crane#43]
OS: Mac OS (M1)
Crane version: 1.3.0
Install method: upgraded from 1.2.1 - replaced app in Applications (also tried deleting and installing fresh)

Steps to reproduce:

Launch Crane
Enter Docker credentials
Click save
Click to the local/remote images lists
Click back to Docker credentials - user/pass are empty

**Comment by kentwelcome** [comment]
```
 ⎈ dev-eks-kent  kent  Applications  Crane.app  …  MacOS  $  ./Crane                                                                                                                                                                             Mon Dec 20 10:44:32 2021
fork process /Applications/Crane.app/Contents/Resources/app.asar/dist/server.js
/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/electron/dist/renderer/backend.js:48
            if (window.__SENTRY_IPC__ == undefined) {
            ^

ReferenceError: window is not defined
    at new RendererBackend (/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/electron/dist/renderer/backend.js:48:13)
    at RendererClient.BaseClient (/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/core/dist/baseclient.js:52:25)
    at new RendererClient (/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/electron/dist/renderer/client.js:19:24)
    at Object.initAndBind (/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/core/dist/sdk.js:18:18)
    at Object.init (/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/electron/dist/renderer/index.js:41:12)
    at Object.specificInit (/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/electron/dist/dispatch.js:121:113)
    at Object.init (/Applications/Crane.app/Contents/Resources/app.asar/node_modules/@sentry/electron/dist/sdk.js:52:16)
    at Object.<anonymous> (/Applications/Crane.app/Contents/Resources/app.asar/dist/ServerHandlers.js:24:8)
    at Module._compile (node:internal/modules/cjs/loader:1110:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1140:10)
    ```

**Comment by aar0nTw** [comment]
pls use `1.2.1` before we fix that

**Comment by aar0nTw** [comment]
solved, pls upgrade to `v1.3.1`
